plugins/experimental/txn_box/plugin/include/txn_box/accl_util.h

```diff
@@ -128,7 +128,7 @@ template <typename Key, typename Value> class StringTree
   /// hold the first element on the entire tree.
   node_type_ptr _head;
   /// this gets incremented on every insert.
-  int32_t _rank_counter;
+  int32_t _rank_counter{0};
```

Comment:
Looks like all this rank stuff may just be kruft:
```
wkaras ~/REPOS/TS/plugins/experimental/txn_box
O$ grep rank $(findsrc)
./plugin/include/txn_box/accl_util.h:    Node(Key k, Value v, int32_t r = -1) : key{k}, value{v}, rank{r}
./plugin/include/txn_box/accl_util.h:    /// key/value rank.
./plugin/include/txn_box/accl_util.h:    int32_t rank;
./plugin/include/txn_box/accl_util.h:  /// TODO: this needs to be changed and use rank to return only 1 value.
./plugin/include/txn_box/accl_util.h:  int32_t _rank_counter{0};
./plugin/include/txn_box/accl_util.h:  // We do not set the rank now, as for the head, -1 should be ok.
./plugin/include/txn_box/accl_util.h:  node_type_ptr new_node = new Node(key, value, _rank_counter++);
./plugin/include/txn_box/accl_util.h:  // For now we return the entire list, we need to work on the rank to send back only one match, the best one.
wkaras ~/REPOS/TS/plugins/experimental/txn_box
O$
```

Reply:
it's not used, but it is there for a `design` reason. It will be used eventually.